### PR TITLE
Update roadmap

### PR DESCRIPTION
**Pull request by mvanlonden**
Adds current objectives and dependency table.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,18 +1,56 @@
...
+## Ongoing Initiatives
+- [ ] Improve documentation, especially for new users to the library
+- [ ] Recipes for “quick start” that people can ideally use/run
+
+
+## Dependent Libraries
+| Repo                                                                         | Release Manager | CODEOWNERS | Pinned     | next/master created | Labels Standardized |
+| ---------------------------------------------------------------------------- | --------------- | ---------- | ---------- | ------------------- | ------------------- |
+| [graphene](https://github.com/graphql-python/graphene)                       | ekampf          | ✅          |            | ✅                   |                     |
+| [graphql-Core](https://github.com/graphql-python/graphql-core)               | Cito            | ✅          |            |                     |                     |
```
Comment (jkimbo) [inline ROADMAP.md]:
```suggestion
| [graphql-core](https://github.com/graphql-python/graphql-core)               | Cito            | ✅          | N/A        | N/A                 |                     |
```

File: ROADMAP.md [hunk 2]
```diff
@@ -1,18 +1,56 @@
...
+| [graphql-core-next](https://github.com/graphql-python/graphql-core-next)     | Cito            | ✅          |            |                     |                     |
```
Comment (jkimbo) [inline ROADMAP.md]:
```suggestion
| [graphql-core-next](https://github.com/graphql-python/graphql-core-next)     | Cito            | ✅          | N/A        | N/A                 |                     |
```

File: ROADMAP.md [hunk 3]
```diff
@@ -1,18 +1,56 @@
...
-- ✨ Improve connections structure
-- 📗 Improve function documentation
-- 🔀 Add support for coroutines in Connection, Mutation (abstracting out Promise requirement) - [See PR](https://github.com/graphql-python/graphene/pull/824)
+### [Core-Next](https://github.com/graphql-python/graphql-core-next)
+Targeted as v3 of [graphql-core](https://pypi.org/project/graphql-core/), Python 3 only
+
+### Graphene
+- [ ] Integrate with the core-next API and resolve all breaking changes
+- [ ] GraphQL types from type annotations - [See issue](https://github.com/graphql-python/graphene/issues/729)
+- [ ] Schema creation from SDL (API TBD)
```
Comment (patrick91) [inline ROADMAP.md]: do we want this, we now have Ariadne and Tartiflette that are schema first libraries for GraphQL, might be worth focusing on other things :)

File: ROADMAP.md [hunk 4]
```diff
@@ -1,18 +1,56 @@
-# Graphene Roadmap
+# GraphQL Python Roadmap
```
Comment (patrick91) [inline ROADMAP.md]: Why this change? :)

File: ROADMAP.md (hunk 3 above)
Comment (jkimbo) [reply ROADMAP.md]: Yeah I agree, I think it would be better to focus time on programmatic schema generation and direct people to other libraries if they want to build their server using the SDL.

File: ROADMAP.md [hunk 5]
```diff
@@ -1,18 +1,56 @@
...
-- 📗 Improve function documentation
-- 🔀 Add support for coroutines in Connection, Mutation (abstracting out Promise requirement) - [See PR](https://github.com/graphql-python/graphene/pull/824)
+### [Core-Next](https://github.com/graphql-python/graphql-core-next)
+Targeted as v3 of [graphql-core](https://pypi.org/project/graphql-core/), Python 3 only
+
+### Graphene
+- [ ] Integrate with the core-next API and resolve all breaking changes
+- [ ] GraphQL types from type annotations - [See issue](https://github.com/graphql-python/graphene/issues/729)
+- [ ] Schema creation from SDL (API TBD)
+- [ ] Improve connections structure
```
Comment (dvndrsn) [inline ROADMAP.md]: What does this point mean? Is there an issue to track "improving connection structure"?

Maybe we should have an issue to track each of these points.

File: ROADMAP.md [hunk 6]
```diff
@@ -1,18 +1,56 @@
...
-- 📄 Schema creation from SDL (API TBD)
-- ✨ Improve connections structure
-- 📗 Improve function documentation
-- 🔀 Add support for coroutines in Connection, Mutation (abstracting out Promise requirement) - [See PR](https://github.com/graphql-python/graphene/pull/824)
+### [Core-Next](https://github.com/graphql-python/graphql-core-next)
+Targeted as v3 of [graphql-core](https://pypi.org/project/graphql-core/), Python 3 only
+
+### Graphene
+- [ ] Integrate with the core-next API and resolve all breaking changes
+- [ ] GraphQL types from type annotations - [See issue](https://github.com/graphql-python/graphene/issues/729)
```
Comment (dvndrsn) [inline ROADMAP.md]: Similar thoughts to @patrik91 and @jkimbo's comments on SDL based schema declaration on this annotation based schema definition.

File: ROADMAP.md (hunk 4 above)
Comment (mvanlonden) [reply ROADMAP.md]: This roadmap touches more than just graphene, want to title to reflect that

File: ROADMAP.md (hunk 3 above)
Comment (mvanlonden) [inline ROADMAP.md]:
```suggestion
```

File: ROADMAP.md (hunk 5 above)
Comment (mvanlonden) [reply ROADMAP.md]: Not totally sure. Copied it over from the old roadmap

File: ROADMAP.md [hunk 7]
```diff
@@ -1,18 +1,55 @@
...
+- [ ] Improve connections structure
```
Comment (mvanlonden) [inline ROADMAP.md]:
```suggestion
```

File: ROADMAP.md (hunk 5 above)
Comment (mvanlonden) [reply ROADMAP.md]: Removed for now

File: ROADMAP.md [hunk 8]
```diff
@@ -1,18 +1,54 @@
...
+### [Core-Next](https://github.com/graphql-python/graphql-core-next)
+Targeted as v3 of [graphql-core](https://pypi.org/project/graphql-core/), Python 3 only
+
+### Graphene
+- [ ] Integrate with the core-next API and resolve all breaking changes
+- [ ] GraphQL types from type annotations - [See issue](https://github.com/graphql-python/graphene/issues/729)
+- [ ] Add support for coroutines in Connection, Mutation (abstracting out Promise requirement) - [See PR](https://github.com/graphql-python/graphene/pull/824)
+
+### Graphene-*
+- [ ] Integrate with the graphene core-next API and resolve all breaking changes
```
Comment (ekampf) [inline ROADMAP.md]: - Apollo Tracing Support
- Improved handling of Python3 enums

**Comment by ekampf** [comment]
oh its already merged :\
